content/ruby/concepts/jruby/jruby.md

```diff
@@ -0,0 +1,32 @@
+---
+Title: 'JRuby'
+Description: 'JRuby is an implementation of Ruby written in Java programming language using the Java Virtual Machine (JVM).'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'JRuby'
+CatalogContent:
+  - 'learn-ruby'
+  - 'learn-rails'
+---
+
```

Comment:
```suggestion
There should be an introductory paragraph without a header. It can be the same as the Metadata description.
```

Reply:
I performed this task in CLI. I will send another PR and link it to #3006